templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+	<h1>Delete manufacturer</h1>
+
+  <p>Are you sure you want to delete the manufacturer: {{ manufacturer }}</p>
+  <p><i>All car with this manufacturer will be also deleted</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input class="btn btn-danger" type="submit" value="Yes, delete">
+    <a class="btn btn-primary" href="{% url 'taxi:car-detail' pk=manufacturer.id %}" role="button">Cancel</a>

```

Comment:
The URL for the cancel button seems incorrect. It currently points to a car detail view. Ensure it points to a relevant manufacturer view, such as the manufacturer list or detail view.